packages/core-commands/src/site-editor-navigation-commands.js

```diff
@@ -136,6 +136,11 @@ const getNavigationCommandLoaderPerPostType = ( postType ) =>
 const getNavigationCommandLoaderPerTemplate = ( templateType ) =>
 	function useNavigationCommandLoader( { search } ) {
 		const history = useHistory();
+		const {
+			params: { path, postType, didAccessPatternsPage },
+		} = useLocation();
```

Comment:
This appears to throw an error when using the command palette in the post editor.

Reply:
We only need to check for these params while in the site editor, so a simple guard against the undefined location might suffice. What do you think?

Reply:
Good spot! Guarding against `undefined` seems logical to me 👍

Reply:
Thank you for finding it! I fixed it with fa499af.